draft-ietf-oauth-status-list.md

```diff
@@ -634,6 +634,10 @@ The Relying Party SHOULD send the following Accept-Header to indicate the reques
 
 If the Relying Party does not send an Accept Header, the response type is assumed to be known implicit or out-of-band.
 
+A successful response that contains a Status List Token MUST use an HTTP status code in the 2xx range. If the Status List Token was not found, the response MUST use the HTTP status code 404.
```

Comment:
```suggestion
A successful response that contains a Status List Token MUST use an HTTP status code in the 2xx range.
```

While I think your right with regard to the majority of implementations here I dont think it pays to call out just 404, otherwise why not other 4xx range codes and or 5xx range codes too?